guide/14-deep-learning/how_retinanet_works.ipynb

```diff
@@ -0,0 +1,159 @@
+{
```

Comment:
<blockquote>This means one would take an image and subsample it into lower resolution and smaller size images (thus, forming a pyramid) and extract hand-engineered features from each layer in the pyramid to detect the objects.&nbsp;</blockquote><p>This means one would take an image and subsample it into lower resolution and smaller size images (thus, forming a pyramid). Hand-engineered features are then extracted from each layer of the pyramid to detect the objects.&nbsp;</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/562/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Better.</p>

Reply:
<p>Better.</p>